src/feature/dirauth/dirvote.c

```diff
@@ -3200,6 +3201,25 @@ dirvote_add_vote(const char *vote_body, const char **msg_out, int *status_out)
     goto err;
   }
 
+  if (!time_posted) { /* I imported this one myself */
+    log_notice(LD_DIR, "Retrieved vote from %s.", vi->address);
```

Comment:
Should we be logging this in all cases where we don't reject the vote?

Reply:
I don't think this log is correct. The vote is either downloaded from another authority, or loaded from disk. The comment "imported" conflicts with the log message "Retrieved", and neither cover both cases.

And I don't think it's helpful to log a notice, we already log lots of info about votes.

I removed it in 15192f88c0.